src/sonic-config-engine/setup.py

```diff
@@ -36,6 +36,7 @@ def get_test_suite():
         'pyyaml',
         'pyangbind==0.6.0',
         'sonic-py-common',
+        'future',
```

Comment:
Please keep this list in alphabetical order.

Reply:
done